app/Finalsite.xcodeproj/project.pbxproj

```diff
@@ -2250,7 +2296,7 @@
 				INFOPLIST_FILE = Jasonette/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.finalsite.admin;
```

Comment:
we should probably keep this out of the codebase if possible, i think both of you changed it

Reply:
lol just saw that you made a similar comment on the other pr

Reply:
yea I think in the past we tried not to change this at all but I accidentally committed the one for the admin POC in here and was setting it back to what the default was in mine